<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>384</width>
    <height>324</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Climate Control</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QGroupBox" name="airFlowDirection">
      <property name="title">
       <string>Air Flow Direction</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QRadioButton" name="rb_FloorPanel">
         <property name="text">
          <string>Floor Panel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="rb_FloorDuct">
         <property name="text">
          <string>Floor Duct</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="rb_BiLevel">
         <property name="text">
          <string>Bi Level</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="rb_DefrostFloor">
         <property name="text">
          <string>Defrost Floor</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="cb_airCondition">
      <property name="text">
       <string>Air Condition</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="cb_heater">
      <property name="text">
       <string>Heater</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="cb_airRecirculation">
      <property name="text">
       <string>Air Recirculation</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QFormLayout" name="formLayout_4">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::FieldsStayAtSizeHint</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="fanSpeedLabel">
        <property name="text">
         <string>Fan Speed</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="sb_fanSpeed"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="steeringWheelHeaterLabel">
        <property name="text">
         <string>Steering Wheel Heater</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="sb_steeringWheelHeater"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
